pkg/kopia/repository/connect_or_create.go

```diff
@@ -19,6 +19,7 @@ import (
 
 	"k8s.io/client-go/kubernetes"
 
+	"github.com/kanisterio/errkit"
 	"github.com/kanisterio/kanister/pkg/kopia/command"
```

Comment:
```suggestion
	"k8s.io/client-go/kubernetes"
	"github.com/kanisterio/errkit"
	
	"github.com/kanisterio/kanister/pkg/kopia/command"
```
errkit package should be considered as external package and put it into separate group